### PR TITLE
added useQuote hook

### PR DESCRIPTION
**Pull request by NancyAanchal**
I tried a bunch of methods but this is the only one that worked. it works but it prints the quote 2 times, I couldn't figure out why.
also moved tajpuriyaquotes.txt to src/quotes/ and renamed it 'tajpuriya.txt' cuz otherwise the path of it from hook would be complex.

### CODE REVIEW

File: nepalingo-web/src/hooks/useNewari.tsx [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
 
+const fetcher = (url: string) =>
+  fetch(import.meta.env.VITE_NEPALBHASA_API_URL + url, {}).then((r) =>
+    r.json()
+  );
 
-const fetcher = (url: string) => fetch(import.meta.env.VITE_NEPALBHASA_API_URL + url, {
-}).then(r => r.json())
+const useNewari = (props: Omit<DictionaryProps, "language">) => {
+  let { data, error, isLoading } = useSWR(
```
Comment (christikaes) [inline nepalingo-web/src/hooks/useNewari.tsx:10]: looks like your branch might be out of date - can you merge in main into this branch (or rebase) ?

File: nepalingo-web/src/quotes/tajpuriya.txt
Comment (christikaes) [inline nepalingo-web/src/quotes/tajpuriya.txt:1]: Can we reformat this into .csv format? I think that will make it easier to edit - eg open in excel etc

**Review by christikaes — CHANGES_REQUESTED** [review]
This looks good! I would just recommend a small change to switch to csv format instead of txt

File: nepalingo-web/src/quotes/tajpuriya.txt
Comment (NancyAanchal) [reply nepalingo-web/src/quotes/tajpuriya.txt:1]: 'small' change? well wouldn't it mean reshaping the whole code 😭? I will try to do that tho!

File: nepalingo-web/src/hooks/useQuotes.tsx [hunk 2]
```diff
@@ -48,14 +45,18 @@ const useQuotes = ({ language }: QuoteProps) => {
...
     if (quotesText) {
       const loadQuotes = () => {
-        const quotesArray = quotesText.split("\r\n\r\n").map((line: string) => {
-          const [text, translation] = line.split("||");
+        const quotesArray = quotesText.split("\n").map((line: string) => {
+          const [text, translation] = line.split(",");
           return {
-            text: text ? text.trim() : "",
-            translation: translation ? translation.trim() : "",
+            text: text ? text.trim().replace(/(^"|"$)/g, "") : "",
```
Comment (christikaes) [inline nepalingo-web/src/hooks/useQuotes.tsx:56]: I don't think you need this anymore right? there are no | separators?

File: nepalingo-web/src/quotes/tajpuriya.txt
Comment (christikaes) [reply nepalingo-web/src/quotes/tajpuriya.txt:1]: you did it! it looks good, just looks like you have a little bit of stale logic here

**Review by christikaes — CHANGES_REQUESTED** [review]
A few small cleanup left

File: nepalingo-web/src/quotes/tajpuriya.txt
Comment (binamkayastha) [reply nepalingo-web/src/quotes/tajpuriya.txt:1]: Yeah our bad, we should have made sure it was a CSV beforehand.

**Comment by christikaes** [comment]
Closing because these changes got merged in the #91 PR

**Pull request closed without merging**